tests/h/services/annotation_json_presentation_test/service_test.py

```diff
@@ -37,14 +29,6 @@ def test_it_configures_formatters(
             formatters.AnnotationUserInfoFormatter.return_value,
         ]
 
-    def test_hidden_status_included_if_user_can_moderate_group(
-        self, formatters, has_permission, svc
-    ):
-        group = mock.Mock()
-        moderator_check = formatters.AnnotationHiddenFormatter.call_args[0][1]
-        moderator_check(group)
-        has_permission.assert_called_once_with("moderate", group)
```

Comment:
?

Reply:
This test in a _very_ roundabout way is trying to test the closured `moderator_check` function which has now gone. Now we just pass the function through without intervention, so there's nothing to test here.